libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/EventBuilderHelper.java

```diff
@@ -122,7 +126,9 @@ private static void createAndStore(String name, UserAccount userAccount, String
         final InstrumentationEventBuilder builder = InstrumentationEventBuilder.getInstance(manager.getAnalyticsManager(),
                                                                                             SalesforceSDKManager.getInstance().getAppContext());
         builder.name(name);
-        builder.startTime(System.currentTimeMillis());
+        builder.startTime(attributes.optLong(START_TIME));
```

Comment:
buildEvent() will use currentTimeMillis if startTime is 0, so no need to provide it here.